SigurdLib.ClientAPI/Localization/LocalizationUtils.cs

```diff
@@ -0,0 +1,44 @@
+using BepInEx;
+using BepInEx.Bootstrap;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace Sigurd.ClientAPI.Localization;
+
+/// <summary>
+/// Provides an easy-to-use localization file registration.
+/// </summary>
+public static class LocalizationUtils
+{
+    /// <summary>
+    /// Register locale files from a specific directory.
+    /// </summary>
+    /// <param name="path">The path of the locale. If left blank, will use (PluginInfo.Location)/locales/(CurrentCulture.Name)</param>
+    /// <returns>A <see cref="Locale"/>.</returns>
+    /// <exception cref="Exception">Thrown when locale path is not found.</exception>
+    public static Locale RegisterLocale(string path = "", string fallbackLanguage = "en-us")
```

Comment:
I don't think 'Register' is an appropriate name for this function - this implies we keep a registry of mods' lang files - perhaps we should, but as-is we don't